src/include/sof/audio/dp_queue.h

```diff
@@ -176,18 +176,6 @@ struct sof_source *dp_queue_get_source(struct dp_queue *dp_queue)
 	return &dp_queue->_source_api;
```

Comment:
I think to avoid unncessary concerns, I'd avoid use of word "backdoor". Somehow a github commit to "add a backdoor" can raise unwanted attention. "Side interface" perhaps?

Reply:
as you see this code is removed by this commit ;)